librz/include/rz_core.h

```diff
@@ -78,6 +78,9 @@ RZ_LIB_VERSION_HEADER(rz_core);
 #define RTR_PROTOCOL_UNIX 4
 
 #define RTR_MAX_HOSTS 255
+#define COLOR_GREEN       "\x1b[32m"
```

Comment:
These are alreaady defined in `librz/include/rz_cons.h`, just remove these lines